Template Files/Template Custom Device File (Inline-Async).xml

```diff
@@ -0,0 +1,131 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
The XML files for most of the custom devices are a mess of tabs, spaces, and other indentation oddities. We don't have to fix this now (feel free to defer), but it would be good to auto-format all of the xml template files so we are starting with a solid foundation.

As is, this file is a mix of spaces and tabs.

Reply:
Yeah, I didn't do anything to the XML template files that were provided to me in the original source. I'll leave it to the VeriStand team to prioritize improving the formatting of the XML files.